WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -0,0 +1,618 @@
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            } else if filter == .Published {
+                alertController.addActionWithTitle(viewButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.viewPost(page)
+                })
+                
+                alertController.addActionWithTitle(draftButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.draftPage(page)
+                })
+                
+                alertController.addActionWithTitle(trashButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            } else {
+                alertController.addActionWithTitle(viewButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+
+                    strongSelf.viewPost(page)
+                })
+                
+                alertController.addActionWithTitle(publishButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.publishPost(page)
+                })
+                
+                alertController.addActionWithTitle(trashButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            }
+        }
+        
+        WPAnalytics.track(.PostListOpenedCellMenu, withProperties: propertiesForAnalytics())
+        
+        if UIDevice.isPad() {
```

Comment:
Fixed the inverted logic.

As for the improvement idea: I've opened [a new ticket to handle that](https://github.com/wordpress-mobile/WordPress-iOS/issues/5220) once this is up.